uint32[2] firsthourupleft
uint32[2] firsthourupright
uint32[2] firsthourbottomleft
uint32[2] firsthourbottomright

uint32[2] secondhourupleft
uint32[2] secondhourupright
uint32[2] secondhourbottomleft
uint32[2] secondhourbottomright

uint32[2] thirdhourupleft
uint32[2] thirdhourupright
uint32[2] thirdhourbottomleft
uint32[2] thirdhourbottomright

uint32[2] firstkgupleft
uint32[2] firstkgupright
uint32[2] firstkgbottomleft
uint32[2] firstkgbottomright

uint32[2] secondkgupleft
uint32[2] secondkgupright
uint32[2] secondkgbottomleft
uint32[2] secondkgbottomright

uint32[2] firsttempupleft
uint32[2] firsttempupright
uint32[2] firsttempbottomleft
uint32[2] firsttempbottomright

uint32[2] secondtempupleft
uint32[2] secondtempupright
uint32[2] secondtempbottomleft
uint32[2] secondtempbottomright

uint32[2] firstcentrupleft
uint32[2] firstcentrupright
uint32[2] firstcentrbottomleft
uint32[2] firstcentrbottomright

uint32[2] secondcentrupleft
uint32[2] secondcentrupright
uint32[2] secondcentrbottomleft
uint32[2] secondcentrbottomright

uint32[2] thirdcentrupleft
uint32[2] thirdcentrupright
uint32[2] thirdcentrbottomleft
uint32[2] thirdcentrbottomright

uint32[2] fourthcentrupleft
uint32[2] fourthcentrupright
uint32[2] fourthcentrbottomleft
uint32[2] fourthcentrbottomright

uint32[2] speedupleft
uint32[2] speedupright
uint32[2] speedbottomleft
uint32[2] speedbottomright



uint32[2] s1upleft
uint32[2] s1upright
uint32[2] s1bottomleft
uint32[2] s1bottomright

uint32[2] s2upleft
uint32[2] s2upright
uint32[2] s2bottomleft
uint32[2] s2bottomright

uint32[2] s3upleft
uint32[2] s3upright
uint32[2] s3bottomleft
uint32[2] s3bottomright

uint32[2] s4upleft
uint32[2] s4upright
uint32[2] s4bottomleft
uint32[2] s4bottomright

uint32[2] s5upleft
uint32[2] s5upright
uint32[2] s5bottomleft
uint32[2] s5bottomright

uint32[2] s6upleft
uint32[2] s6upright
uint32[2] s6bottomleft
uint32[2] s6bottomright

uint32[2] s7upleft
uint32[2] s7upright
uint32[2] s7bottomleft
uint32[2] s7bottomright

uint32[2] s8upleft
uint32[2] s8upright
uint32[2] s8bottomleft
uint32[2] s8bottomright

uint32[2] s9upleft
uint32[2] s9upright
uint32[2] s9bottomleft
uint32[2] s9bottomright

uint32[2] s10upleft
uint32[2] s10upright
uint32[2] s10bottomleft
uint32[2] s10bottomright

uint32[2] s11upleft
uint32[2] s11upright
uint32[2] s11bottomleft
uint32[2] s11bottomright

uint32[2] s12upleft
uint32[2] s12upright
uint32[2] s12bottomleft
uint32[2] s12bottomright

uint32[2] s13upleft
uint32[2] s13upright
uint32[2] s13bottomleft
uint32[2] s13bottomright

uint32[2] prg1left
uint32[2] prg1right

uint32[2] prg2left
uint32[2] prg2right

uint32[2] prg3left 
uint32[2] prg3right

uint32[2] prg4left
uint32[2] prg4right

uint32[2] prg5left
uint32[2] prg5right

uint32[2] prg6left
uint32[2] prg6right

uint32[2] prg7left
uint32[2] prg7right

uint32[2] prg8left
uint32[2] prg8right

uint32[2] prg9left
uint32[2] prg9right

uint32[2] prg10left
uint32[2] prg10right

uint32[2] prg11left
uint32[2] prg11right

uint32[2] option1left 
uint32[2] option1right

uint32[2] option2left
uint32[2] option2right

uint32[2] option3left
uint32[2] option3right

uint32[2] option4left
uint32[2] option4right





